docker-compose.yml

```diff
@@ -102,7 +102,7 @@ services:
 
   web:
     image: warehouse:docker-compose
-    command: gunicorn --reload -b 0.0.0.0:8000 --access-logfile - --error-logfile - warehouse.wsgi:application
+    command: gunicorn --reload --reload-extra-file=warehouse/api/openapi.yaml -b 0.0.0.0:8000 --access-logfile - --error-logfile - warehouse.wsgi:application
```

Comment:
This was a cool hack, and I've found that I probably want to use `hupper` instead, since the entire gunicorn process exits on a failure to start up with a broken openapi example.

Reply:
hupper isn't the right answer here. I don't know what is, since "gunicorn failing to boot the worker, so master process shuts down" seems like the general right answer.

Reply:
Adding `restart: on-failure` to my `web` worker in my docker compose override is what works best. if it crashes, it'll restart. yes, that will be an infinite loopp on my machine, but that works well enough for my local dev purposes.